supervisors/sensitivity_analysis_morris/scalarmapi.R

```diff
@@ -0,0 +1,106 @@
+setGeneric("check_repetitions", function(.Object, params) {
+  .Object
+})
+setGeneric("schedule_point", function(.Object, params) {
+  .Object
+})
+setGeneric("get_result", function(.Object, params) {
+  .Object
+})
+setGeneric("mark_as_complete", function(.Object, result) {
+  .Object
+})
+Scalarm = setClass("Scalarm",
```

Comment:
@mwrona, @Zuchens - OK, z tego co widzę poprawione :)

Tak nawiasem, jeśli chodzi o fragment HAML:
```
%h3.subheader= 'Sensitivity Analysis'
```
To widzę, że jest poprawiony, ale nawet ja stosuję tutaj "klejenie" = do lewej, bo zdaje się, że tak mamy w większości kodu i też tak widziałem (https://rivalfox.com/blog/haml-best-practices/). Wydawało mi się, że w tym przypadku spacja powodowała problemy - to działa?